### DISCUSSION TITLE
hactribe as external sequencer?

### DISCUSSION THREAD

**Discussion by VasileiosLiaskas** [bangcorrupt/hacktribe#260] [Q&A] [answered]
Can hacktribe sequence other synths via midi?

**Comment by enorrmann** [comment, marked as answer]
yes, each pad transmits on its own channel, from 1 to 16
edit: to clarify, the electribe alone can do this, no need for hacktribe

**Reply by bangcorrupt** [reply]
In the future there should be more features for sequencing external gear.  The [experimental branch](https://github.com/bangcorrupt/hacktribe/tree/experimental) has MIDI soft thru and sends motion sequences as CC messages.  It's buggy and offers no configuration, but it works.
